docker-compose.yml

```diff
@@ -17,7 +17,11 @@ services:
         container_name: dcsa_db
         environment:
             POSTGRES_PASSWORD: "change_this_password"
-        build:
-            DCSA-Information-Model/datamodel/
+        image: postgres:10-alpine
         ports:
             - 5432:5432
+        volumes:
+          - ./DCSA-Information-Model/datamodel/initdb.d:/docker-entrypoint-initdb.d:ro
+          - ./DCSA-Information-Model/datamodel/referencedata.d:/referencedata.d:ro
+          - ./DCSA-Information-Model/datamodel/samples.d:/samples.d:ro
+          - ./DCSA-Information-Model/datamodel/testdata.d:/testdata.d:ro
```

Comment:
I think this should go into the docker file in the IM.

Reply:
I prefer to avoid making a "special" database image just for some tables and some data.

Reply:
The docker file already includes the first 3, so they are redundant.  As for the last one, we can also load that externally like we do in all the other CIs.

Reply:
They would be redundant if I was using the IM image. But this PR changes that to use a pure postgresql image so we don't have to build and rebuild the image all the time.